packages/components/pie-modal/src/index.ts

```diff
@@ -144,40 +157,47 @@ export class PieModal extends RtlMixin(LitElement) {
     }
 
     /**
-     * Dismisses the modal on backdrop click
+     * Dismisses the modal on backdrop click. (dependent on prop: `isDismissible`)
      *
      */
     private _handleDialogLightDismiss = (event: MouseEvent) : void => {
-        const rect = this._dialog?.getBoundingClientRect();
+        if (this.isDismissible) {
```

Comment:
Keen to hear your thoughts on this - would changing this to simply do an early return if `isDismissible` is false be easier to read in the future?

That way you know you don't need to read ahead for any additional logic.

Reply:
fixed.